index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dany's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+                <a href="#"class="Logo">My Logo</a>
+                <nav class="Navbar">
+                    <div class="menu"><a href="#"><img src="img/humburger-btn.svg" alt="menu"></a></div>
+                </nav>
+                
+    </header>
+    <section>
+            <h1 class="perso">My name is Dany<br> I am glad to meet you!</h1>
+            <p class="pro"> I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+            <h2 class="connect">LET'S CONNECT</h2>
+            <ul class="icon">
+                <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
+                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
+                <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
```

Comment:
- [ ] I am so grateful that you were able to push the images also. Though we are currently missing 2 icons. Kindly ensure to add them so that your design matches the Figma design as much as possible ☺️ .
- [ ] [OPTIONAL] In addition, you should ensure that the `ul` HTML tag is closed.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219882496-d589b201-f89c-4bc4-a735-0daa9e98ce5d.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219882547-4a7db339-c375-4b3a-828c-eed9613db933.png)